vertical-pod-autoscaler/pkg/admission-controller/resource/pod/recommendation/recommendation_provider.go

```diff
@@ -56,10 +56,10 @@ func GetContainersResources(pod *core.Pod, vpaResourcePolicy *vpa_types.PodResou
 		recommendation := vpa_api_util.GetRecommendationForContainer(container.Name, &podRecommendation)
 		if recommendation == nil {
 			if !addAll {
-				klog.V(2).Infof("no matching recommendation found for container %s, skipping", container.Name)
+				klog.V(2).InfoS("No recommendation found for container, skipping", "container", container.Name)
 				continue
 			}
-			klog.V(2).Infof("no matching recommendation found for container %s, using Pod request", container.Name)
+			klog.V(2).InfoS("No matching found for container, using Pod request", "container", container.Name)
```

Comment:
The wording of "No matching found for container" seems strange, is this better:
```suggestion
			klog.V(2).InfoS("No match found for container, using Pod request", "container", container.Name)
```